src/libs/antares/study/scenario-builder/rules.cpp

```diff
@@ -162,7 +162,7 @@ void Rules::loadFromInstrs(Study& study,
         }
     }
 
-    if (kind_of_scenario == "r")
+    if (kind_of_scenario == "r" && study.parameters.renewableGeneration() == rgClusters)
```

Comment:
use **study.parameters.renewableGeneration.isClusters()** instead ?

Reply:
done

Reply:
Maybe we could remove all occurences of **study.parameters.renewableGeneration() == rgClusters or rgAggregated** and replace them with **study.parameters.renewableGeneration.isClusters()** or **study.parameters.renewableGeneration.isAggregated()**, so that there would be no more reference to enum **RenewableGenerationModelling** in these files and possibly we could remove headers inclusion